### ISSUE TITLE
improve documentation on how to use

### ISSUE THREAD

**Issue by jpetrucciani** [jpetrucciani/nix#1444]
right now, i have some examples in the `examples/` directory - but it would be nice to have some example code to grab for people to be able to use this repo as a flake input or from general nix files

**Comment by Animeshz** [comment]
Yeah, I'm a bit confused about the `hex` directory, I'd like to see how its used, unlike pog which seems like compiles into shell script and can be utilized as commands the hex's purpose seems unclear to me.

Thanks in advance!

**Comment by Animeshz** [comment]
Also, not the best place to ask, but if you do recurrently refer to some other people's config for reference, please link them to README, because nix docs have always been not so good, and you're style of organizing nix files looks fabulous for real! Must have been creative mind and lot of research on nix.

**Comment by jpetrucciani** [comment]
Sorry about the delay! I've updated some stuff here, but will continue to add/improve it in the coming few days/weeks.

> Yeah, I'm a bit confused about the hex directory, I'd like to see how its used, unlike pog which seems like compiles into shell script and can be utilized as commands the hex's purpose seems unclear to me.

I've added a [few notes about what it can be used for, and a very basic example](https://github.com/jpetrucciani/nix/tree/main/examples/hex). I will add some more shortly!

> but if you do recurrently refer to some other people's config for reference, please link them to README

I've updated the main README a bunch today, to better explain some stuff, link to more places, and add some nice resources! I will link a few more things down there soon

**Comment by jpetrucciani** [comment]
Just updating a bit here - but i've been working on splitting out `pog` and `hex` into their own, standalone repos, with better documentation in #1445 and #1446. Pog is now usable from its own import (and also supports flakes) in [jpetrucciani/pog](https://github.com/jpetrucciani/pog) - I am also working on better docs, published to github pages for that. I am working on the same with `hex` that should be pushed later this week!
